settings.gradle

```
@@ -1,8 +1,16 @@
 pluginManagement {
     plugins {
         id 'org.jetbrains.kotlin.android' version '1.4.31'
+        id "com.automattic.android.publish-to-s3" version "0.7.0"
     }
     repositories {
+        maven {
+            url 'https://a8c-libs.s3.amazonaws.com/android' 
+            content {
+                includeGroup "com.automattic.android"
+                includeGroup "com.automattic.android.publish-to-s3"
```

Another annoying nitpick about quotes usage

```suggestion
                includeGroup 'com.automattic.android'
                includeGroup 'com.automattic.android.publish-to-s3'
```